pkg/xray/util.go

```diff
@@ -126,6 +126,13 @@ func copyStringMap(source map[string]string, target map[string]string) map[strin
 	return target
 }
 
+func copyInterfaceMap(source map[string]interface{}, target map[string]interface{}) map[string]interface{} {
```

Comment:
use `(dest map[string]interface{}, sources ... map[string]interface{})` and have this work on any number of maps

Reply:
removed completely, I don't use it